readme.txt

```diff
@@ -122,6 +104,9 @@ To see a list of all the premium extensions and learn more visit [http://wiredim
 
 == Changelog ==
 
+= 1.1.15 =
+* Removed all references to premium extensions.
```

Comment:
What about updating this to be "Removed all references to premium extensions that will no longer be for sale." to make it more clear?

Reply:
Ah yes, that sounds much better.